waddrmgr/db.go

```diff
@@ -112,9 +132,16 @@ type dbChainAddressRow struct {
 	index  uint32
 }
 
-// dbImportedAddressRow houses additional information stored about an imported
+// dbImportedP2PKHAddressRow houses additional information stored about an imported
+// P2PKH address in the database.
+type dbImportedP2PKHAddressRow struct {
```

Comment:
I think we should allow importing watching P2SH addresses as well, without the associated script to redeem the output.  Rename this to dbImportedHash160AddressRow?